src/app/grid/grid-sample-selection-template-numbers/grid-sample-selection-template-numbers.component.scss

```diff
@@ -0,0 +1,43 @@
+.grid__wrapper {
+    margin: 0 16px;
+}
+
+.header-selector {
+    width: 70px;
+    height: 60px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.header-image {
+    width: 40px;
+    height: 40px;
+}
+
+.row-selector {
+    width: 70px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 0 10px;
+    height: 100%;
+}
+
+.row-selector-checkbox {
+    pointer-events: none;
```

Comment:
Not sure what that is for.. does `disableRipple` on the checkbox not work?

Reply:
leftover used before we made checkbox readonly :)